app/assets/scripts/views/imagery-search.js

```diff
@@ -130,6 +165,25 @@ var ImagerySearch = React.createClass({
     }
   },
 
+  onMapChange: function () {
+    if (this.mapMovementThrottle) return;
+    // Using Browser History just to update the hash URL. React's hashHistory causes
+    // a scroll back to the top of the page. hashHistory is still a dependency because
+    // of the expectation that the host does not support SPA routing to index.html.
+    const path = `#/imagery-search/${this.mapViewToString()}`;
+    window.history.replaceState(null, null, path);
```

Comment:
The scroll to the top is not related to hashHistory. It's an intentional plugin that fires when the url changes. This is needed for when we change to pages with a lot of content. It's implemented [here](https://github.com/hotosm/imagery-coordination/blob/develop/app/assets/scripts/main.js#L51-L53).

I don't have a better suggestion about this, besides adding a condition for that specific path to the scroll function, so this is more a fyi.